documenters_aggregator/spiders/parkdistrict.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+import re
+import scrapy
+
+from datetime import datetime
+from pytz import timezone
+from legistar.events import LegistarEventsScraper
+
+
+class ParkdistrictSpider(scrapy.Spider):
+    name = 'parkdistrict'
+    long_name = 'Chicago Park District'
+    START_URL = 'https://chicagoparkdistrict.legistar.com'
+    allowed_domains = ['chicagoparkdistrict.legistar.com']
+    start_urls = [START_URL]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        events = self._make_legistar_call()
+        return self._parse_events(events)
+
+    def _make_legistar_call(self, since=None):
+        les = LegistarEventsScraper(jurisdiction=None, datadir=None)
+        les.EVENTSPAGE = self.START_URL + '/Calendar.aspx'
+        les.BASE_URL = self.START_URL
+        if not since:
+            since = datetime.today().year
+        return les.events(since=since)
+
+    def _parse_events(self, events):
+        for item, _ in events:
+            data = {
+                '_type': 'event',
+                'id': self._parse_id(item),
+                'name': self._parse_name(item),
+                'description': self._parse_description(item),
+                'classification': self._parse_classification(item),
+                'start_time': self._parse_start(item),
+                'end_time': self._parse_end(item),
+                'all_day': self._parse_all_day(item),
+                'location': self._parse_location(item),
+                'sources': self._parse_sources(item)
+            }
+            data['status'] = self._parse_status(item, data['start_time'])
+            yield data
+
+    def _parse_id(self, item):
+        """
+        Calulate ID. ID must be unique within the data source being scraped.
+        """
+        new_id = item['Name'] + item['Meeting Date']
+        return ''.join(ch for ch in new_id if ch.isalnum())
+
+    def _parse_classification(self, item):
+        """
+        Parse or generate classification (e.g. town hall).
+        """
+        return 'Not classified'
+
+    def _parse_status(self, item, start_time):
+        """
+        passed = meeting already started
+        tentative = no agenda posted
+        confirmed = agenda posted
+        """
+        if datetime.now().isoformat() > start_time:
```

Comment:
I am a little worried about using `status` to indicate two different things: whether a meeting has already started and if an agenda has been posted. Using the presence of an agenda to determine if a meeting is confirmed is clever but not totally obvious to those consuming the data.

I am OK with merging this as is, though, while we think about how to handle this better. I think we probably ought to have a place for an agenda URL in the event schema. Additionally, whether the meeting has started of now can be inferred from the `start_time`, so altering this value based the current time doesn't provide a lot of value and creates churn in the data.

I have created #169 to track making those schema changes, at which point we can come back and update this spider.